modules/ducktests/tests/ignitetest/tests/smoke_test.py

```diff
@@ -87,3 +87,28 @@ def test_zk_start_stop(self):
         zookeeper = ZookeeperService(self.test_context, num_nodes=3)
         zookeeper.start()
         zookeeper.stop()
+
+    @cluster(num_nodes=4)
+    @ignite_versions(str(DEV_BRANCH))
+    def test_ignite_clock_sync(self, ignite_version):
+        """
+        Tests that clocks are synchronized between nodes.
+        """
+        num_nodes = 4
```

Comment:
Why do you check only 4 nodes. We should check that every node in a cluster is configured correctly.

Reply:
Fixed